src/test/java/com/marklogic/spark/reader/optic/PushDownParamTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2024 MarkLogic Corporation. All Rights Reserved.
+ */
+package com.marklogic.spark.reader.optic;
+
+import com.marklogic.spark.Options;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PushDownParamTest extends AbstractPushDownTest {
+    static final String LEXICON_BIND_QUERY = wrapNonViewQuery(
+    "op.fromLexicons("
```

Comment:
The connector doesn't have a hard requirement on `op.fromView`, but that's the only accessor that will scale as that's the only one we can support partitions for. 

In the Optic WG, the consensus was that parameterized CTS queries should be supported by the server, but not invented in the Spark connector. 

Is it possible to produce a test that uses `op.fromView`? I'd still consider it as an addition to the connector, but I'm generally loath to add anything that really ought to be handled by the server.

Reply:
Updated with simpler test cases that use op.fromView instead.

I agree that parameterized CTS queries should be core-server functionality.  This branch isn't about parameterizing CTS queries, but about exposing the Optic parameter bindings that already exist and are handled by the core server to downstream consumers.  Spark doesn't have a concept of parameter bindings, and neither does ODBC (which many downstream consumers use through Spark via Thrift/Hive).  Optic bindings have many use cases - term expansion, type casting, allowing cached query plans, and more.

This branch is one approach at mapping this core server functionality onto Spark's capabilities in a way that is declarative and opt-in from a Spark developer perspective, while still being agnostic and transparent from a downstream user perspective.

In addition, even if/when parameterized CTS queries are implemented by the core server, we will still need similar functionality to expose it to Spark.  This approach can be expanded in the future to support that when the capability is there.  (I will still create a PoC for it as I need it for development/exploration purposes, but that will be on a different branch and won't have a PR until/unless the core server supports it directly).